pkg/apis/batch/types.go

```diff
@@ -44,13 +44,23 @@ const (
 	JobNameLabel = labelPrefix + LegacyJobNameLabel
 	// Controller UID is used for selectors and labels for jobs
 	ControllerUidLabel = labelPrefix + LegacyControllerUidLabel
+	// Label indicating the controller that manages a Job. Jobs with a custom value
+	// of this label - any label value other than `job-controller.k8s.io` - are
```

Comment:
Use active voice: The k8s job controller skips Jobs with a custom value for this label

Reply:
Done, I have also rephrased a little to make it shorter. PTAL.